### ISSUE TITLE
[Suggestion] "Insulated" Block of Compressed Iron

### ISSUE THREAD

**Issue by Fayti1703** [MineMaarten/PneumaticCraft#599]
I'd like to suggest adding a "Insulated" Block of Compressed Iron, which would look just like a normal Block of Compressed Iron, but it would not get affected by Temperature. This would be useful if you wanted to build a structure with the Blocks, as it would reduce the amount of tile entities and make the Blocks actually look like they do in the inventory.

**Comment by Forecaster** [comment]
Can the block be turned into a cover of facade or similar to be used for decoration?

**Comment by CaptiveCreeper** [comment]
Yes if you have applied energistics installed you can make a facade out the compressed iron block
